elements/lisk-p2p/src/p2p.ts

```diff
@@ -115,9 +115,11 @@ export const EVENT_NEW_PEER = 'newPeer';
 export const NODE_HOST_IP = '0.0.0.0';
 export const DEFAULT_DISCOVERY_INTERVAL = 30000;
 export const DEFAULT_BAN_TIME = 86400;
+export const DEFAULT_POPULATOR_INTERVAL = 10000;
```

Comment:
Is this interval specified in LIP somewhere?

Reply:
No, it's not. We can decide what to set by default.